docs/logging.md

```diff
@@ -0,0 +1,59 @@
+# Logging
+
+Using Bigflow you can easily add handlers to your loggers that will send all loggers calls to the Cloud Logging.
+In the CL you can filter logs per project id/logger name and workflow id. 
```

Comment:
nit: it is easier to read full 'Cloud Logging' instead of CL.

Reply:
I prefer to stay with CL